sigs.yaml

```diff
@@ -1,4 +1,26 @@
-sigs: []
+sigs:
+- dir: sig/sig-dataScience
```

Comment:
please remove "sig/" from the dir and use dash-case dirname

Reply:
I'm happy to change the naming, but I was following the example of the existing SIG dirs;
![image](https://user-images.githubusercontent.com/4494906/148814460-f251a2b3-a60b-4bdc-aaba-06ad6c0c422a.png)
should they all be modified to dash-case. 

Also, should we remove the "sig/"? In the kubernetes example the sig directories are in the root of the project, but we have them one level below in "sig/" so don't we need this?

Reply:
> should they all be modified to dash-case.
ja.

> Also, should we remove the "sig/"? 
ja.

:)